shared/macros-bash.jinja

```diff
@@ -53,8 +53,8 @@
 # rc: The error code (optional, default is 1)
 # action: What to do (optional, default is 'exit', can be also 'return' or anything else)
 #}}
-{{% macro die(message, rc=1, action="exit") -%}}
-printf '%s\n' '{{{ message }}}' >&2
+{{% macro die(message, rc=1, action="exit") %}}
```

Comment:
Is the removal of `-` before `%` intended?

Reply:
No, but other dashes had to be added elsewhere :-)